ceph/rados/objectstoretool_workflows.py

```diff
@@ -33,7 +33,7 @@ class objectstoreToolWorkflows:
     Contains various functions to verify ceph-objectstore-tool commands
     """
 
-    def __init__(self, node: CephAdmin):
+    def __init__(self, node: CephAdmin, nostop=None, nostart=None):
```

Comment:
please add a safety check such that both nostop & nostart vars cannot be sent as true for a single object.